docs/source/foundry/models.mdx

```diff
@@ -0,0 +1,29 @@
+# Supported Models
+
+Around +11,000 open models from the Hugging Face Hub are made available on Microsoft Foundry and Azure Machine Learning, which represents a subset out of the +1,800,000 public open-models, as the Hugging Face collection in Azure is a curated subset of the most downloaded / relevant models on the Hub, as well as compatible with Transformers, Sentence Transformers and Diffusers, as well as with other Hugging Face libraries and solutions.
+
+<Tip>
+
+Even if you don't have a Microsoft Azure account, you can still explore the [public Hugging Face Collection on Microsoft Foundry](https://ai.azure.com/catalog/publishers/hugging%20face,huggingface).
+
+</Tip>
+
+This being said, the supported models range different architectures and backends, but a way to identify whether a model from the Hugging Face Hub is made available within the model catalog in Microsoft Foundry and Azure Machine Learning is to either:
+
+1. Navigate to the model card of the given model under https://huggingface.co/models, and make sure that the "Deploy" button is made available and that the "Deploy on Microsoft Foundry" option is listed there. If that's the case, then if the model is available on Microsoft Foundry, the URL pointing to the model on Microsoft Foundry will be provided via the "Go to model on Microsoft Foundry" button, otherwise, the button "Request to add" will show to request the model addition (more information on the latter in [Request a model addition in the Hugging Face collection on Microsoft Foundry](../guides/request-model-addition)).
```

Comment:
```suggestion
1. Navigate to the model card of the given model under https://huggingface.co/models, and check whether the "Deploy" button is available with the "Deploy on Microsoft Foundry" option listed there. If available, the URL pointing to the model on Microsoft Foundry will be provided via the "Go to model on Microsoft Foundry" button. Otherwise, you can request the model's addition by clicking "Request to add" (See [Request a model addition in the Hugging Face collection on Microsoft Foundry](../guides/request-model-addition) for details).
```